src/main/host/descriptor/mod.rs

```diff
@@ -168,13 +168,13 @@ impl Drop for LegacyListener {
 
 /// Stores event listener handles so that `c::StatusListener` objects can subscribe to events.
 struct LegacyListenerHelper {
-    handle_map: std::collections::HashMap<usize, Handle<(FileState, FileState)>>,
+    handles: Vec<(usize, Handle<(FileState, FileState)>)>,
```

Comment:
A comment here about what the `usize` represents would be helpful.

Reply:
Added a comment.